packages/melody-plugin-jsx/__tests__/__snapshots__/CompilerSpec.js.snap

```diff
@@ -172,7 +172,7 @@ _template.render = function (_context) {
             }} /></li><li>{temp(strtotime(\\"+1 day\\", \\"2018-03-01\\"))(\\"Y-m-d\\")}</li><li>{_context.elephant != null && _context.elephant !== '' ? _context.elephant : \\"elephant\\"}</li><li>{escape(\\"<div>HTML</div>\\")}</li><li>{first(myArray)}</li><li>{format(\\"I like %s, it is %s\\", \\"Twig\\", \\"nice\\")}</li><li>{[1, 2, 3].join(\\"|\\")}</li><li><span dangerouslySetInnerHTML={{
                 __html: JSON.stringify(myData)
             }} /></li><li>{keys(myData)}</li><li>{last(myArray)}</li><li>{myArray.length}</li><li>{\\"Lower\\".toLowerCase()}</li><li>{merge(myArray, [\\"Yes\\"]).length}</li><li><span dangerouslySetInnerHTML={{
-                __html: nl2br(\\"nThis is a new line\\\\\\\\nAnd this\\")
+                __html: nl2br(\\"\\\\\\\\nThis is a new line\\\\\\\\nAnd this\\")
```

Comment:
This is a snapshot that I did not intend to change. However, I think it rather fixes it than breaks it, because, with the first group of backslashes stripped (as it was before), the lonely "n" at the beginning certainly does not look right.